packages/terra-popup/src/terra-dev-site/doc/example/PopupNoHeader.scss

```diff
@@ -0,0 +1,8 @@
+:local {
```

Comment:
This file is same code as popupbounded.scss. can we resuse them by renaming the file ?

Reply:
I agree, i have renamed and reused PopupBounded.scss and removed PopupNoHeader.scss.